public/static/docs/command-reference/repro.md

```diff
@@ -45,6 +45,39 @@ files, intermediate or final results. It saves all the data files, intermediate
 or final results into the <abbr>DVC cache</abbr> (unless `--no-commit` option is
 specified), and updates stage files with the new checksum information.
 
+### Parallel stage execution
+
+Currently, `dvc repro` is not able to parallelize stage execution automatically.
+If you need to do this, you can launch `dvc repro` multiple times manually.
+
+For example, let's say a <abbr>pipeline</abbr> graph looks something like this:
```

Comment:
I've already approved this but if you want you could combine these 2 sentences now that the paragraph above has become pretty short. No strong preference either way, just a note. Thanks again 🙂

Reply:
Good one. Done.